static/css/main.css

```diff
@@ -25,6 +25,7 @@ h1 {
 
 .org-image {
```

Comment:
Rule doesn't have all its properties in alphabetical order. (order-alphabetical)

*Origin: CSSLintBear, Section: `css`.*